daffodil-japi/src/main/scala/org/apache/daffodil/japi/infoset/Infoset.scala

```diff
@@ -454,21 +211,21 @@ abstract class InfosetInputterProxy extends InfosetInputter {
 /**
  * A proxy for InfosetOutputters that are internal to Daffodil
  */
-abstract class InfosetOutputterProxy extends InfosetOutputter {
```

Comment:
Don't need this now, but will probably again when JAPI gets put back together with proxying objects.